portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,14 +36,31 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Maybe add a code comment here, explaining briefly what this block of logic is intended to do. That's usually a good engineering practice no matter the coding language.